Utils/Functions/Net/GithubReleases/script.js

```diff
@@ -27,4 +27,4 @@ function getGithubVersions(repositoryOwner, repositoryName, wizard) {
 
     return versions;
 }
-module.getGithubVersions = getGithubVersions;
+module.getGithubReleases = getGithubReleases;
```

Comment:
If the method does not call itself (recursively) it is ok to directly export it, i.e.:

```javascript
module.getGithubReleases = function getGithubReleases(repositoryOwner, repositoryName, wizard) {
   ...
};
```

Reply:
I did this like in the other utils function (ls, remove, ...)

Reply:
Yes I split it there in two parts (the function definition and the `module` assignment) because `touch` uses `fileExists` which does not work in the same module if `fileExists` is directly exported. (You would need to use `module.fileExists()` in this case which is much harder to read)